src/tracker.jl

```diff
@@ -0,0 +1,34 @@
+abstract type Tracker end
+
+function init_tracker(tracker::Tracker, image::Array{T, 2}, bounding_box::MVector{4, Int}) where T
+    @assert bounding_box[1] >= 1 && bounding_box[1] <= size(image)[1]
```

Comment:
Personally, I use `@assert` mostly for quick debug and development purpose, when things become stable, I would then throw more descriptive errors (e.g., `ArgumentError`) to replace the simple `AssertionError`.

Reply:
Thanks! I also think that it will be good to throw more descriptive errors when things become stable.